src/index.js

```diff
@@ -459,6 +481,7 @@ const stacLayer = async (data, options = {}) => {
     // if we still haven't found a valid imagery layer yet, just add the first COG
     const cogs = Object.entries(assets).filter(entry => isAssetCOG(entry[1]));
     if (!addedImagery && cogs.length >= 1) {
+      console.log('adding a COG')
```

Comment:
I assume this was accidentally left in, but if you want to log something here, I'd recommend mimicking the pattern used elsewhere with debug level and prefix in the message:
```js
if (debugLevel >= 1) console.log("[stac-layer] ...");
```

Reply:
removed :)